src/chef-server-ctl/bin/chef-server-ctl

```diff
@@ -75,13 +76,13 @@ module Omnibus
           last = exec_list.shift
           connection.exec(last)
         end
-        puts "Chef Server databases and roles have been successsfully deleted from #{postgres["vip"]}"
+        puts "#{ChefUtils::Dist::Server::PRODUCT} databases and roles have been successfully deleted from #{postgres['vip']}"
```

Comment:
Nice! It should be Chef Infra Server not Chef Server. (ref: https://github.com/chef/chef/blob/main/chef-utils/lib/chef-utils/dist.rb#L103)